.happy/terraform/modules/sfn_config/nextstrain.wdl

```diff
@@ -115,7 +116,7 @@ task nextstrain_workflow {
         --phylo-run-id "${workflow_id}"                                         \
         --bucket "${aspen_s3_db_bucket}"                                        \
         --key "${key}"                                                          \
-        --tree-path /ncov/auspice/ncov_aspen.json                               \
+        --tree-path ncov/auspice/ncov_aspen.json                                \
     >>>
```

Comment:
is this a typo from a merge conflict?

Reply:
i did something wrong and merge conflict showed up TnT i removed this in `20acd2c`

Reply:
oh no looks like this is still here?

Reply:
maybe you just removed it in the other file?

Reply:
The >>> is necessary! it signals the end of a bash block in wdl. Example: https://github.com/chanzuckerberg/aspen/blob/trunk/.happy/terraform/modules/sfn_config/gisaid.wdl#L101

Reply:
ohhhhhhhhh thanks for jumping in on that @jgadling!! Sorry @danrlu for the misguidance!!!

Reply:
@jgadling would you mind reviewing/approving this? I may not have the correct context here to give a good review

Reply:
Thanks Jess and Phoenix <3 It's my fault not to look carefully. Very good learning though!